src/traces/scattermapbox/defaults.js

```diff
@@ -28,6 +28,15 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         return;
     }
 
+    var clusterEnabled = coerce('cluster.enabled');
+    if(clusterEnabled) {
+        coerce('cluster.maxzoom');
+        coerce('cluster.radius');
+        coerce('cluster.step');
+        coerce('cluster.color', defaultColor);
+        coerce('cluster.size');
```

Comment:
Let's also consider using `marker.color` and `marker.size` as the `default` second argument passed to `coerce`.
For that we need to move this block below `coerce(marker.`*.

Something like:
```js
var marker;
if(subTypes.hasMarkers(traceOut)) {
    handleMarkerDefaults(traceIn, traceOut, defaultColor, layout, coerce, {noLine: true});
    coerce('marker.allowoverlap');
    coerce('marker.angle');
    // array marker.size and marker.color are only supported with circles
    marker = traceOut.marker;
    if(marker.symbol !== 'circle') {
        if(Lib.isArrayOrTypedArray(marker.size)) marker.size = marker.size[0];
        if(Lib.isArrayOrTypedArray(marker.color)) marker.color = marker.color[0];
    }
}

var clusterEnabled = coerce('cluster.enabled');
if(clusterEnabled) {
    coerce('cluster.color', marker.color || defaultColor);
    var clusterSize = coerce('cluster.size', marker.size);
    coerce('cluster.radius', clusterSize);
    coerce('cluster.maxzoom');
    coerce('cluster.step'); // if necessary
}
```

Reply:
Above comment is not addressed at the moment.